### PR TITLE
PascalABCNET.PascalABCNET-stable version 3.8.2.3074

### PR DESCRIPTION
**Pull request by BareGamer**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/50142)

### CODE REVIEW

File: manifests/p/PascalABCNET/PascalABCNET-stable/3.8.2.3074/PascalABCNET.PascalABCNET-stable.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: PascalABCNET.PascalABCNET-stable
```
Comment (Trenly) [inline manifests/p/PascalABCNET/PascalABCNET-stable/3.8.2.3074/PascalABCNET.PascalABCNET-stable.installer.yaml:4]:
```suggestion
PackageIdentifier: PascalABCNET.PascalABCNET
```
Stable isn't needed here as the primary channel

File: manifests/p/PascalABCNET/PascalABCNET-stable/3.8.2.3074/PascalABCNET.PascalABCNET-stable.locale.ru-Ru.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: PascalABCNET.PascalABCNET-stable
```
Comment (Trenly) [inline manifests/p/PascalABCNET/PascalABCNET-stable/3.8.2.3074/PascalABCNET.PascalABCNET-stable.locale.ru-Ru.yaml:4]:
```suggestion
PackageIdentifier: PascalABCNET.PascalABCNET
```

File: manifests/p/PascalABCNET/PascalABCNET-stable/3.8.2.3074/PascalABCNET.PascalABCNET-stable.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: PascalABCNET.PascalABCNET-stable
```
Comment (Trenly) [inline manifests/p/PascalABCNET/PascalABCNET-stable/3.8.2.3074/PascalABCNET.PascalABCNET-stable.yaml:4]:
```suggestion
PackageIdentifier: PascalABCNET.PascalABCNET
```

File: manifests/p/PascalABCNET/PascalABCNET-stable/3.8.2.3074/PascalABCNET.PascalABCNET-stable.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: PascalABCNET.PascalABCNET-stable
+PackageVersion: 3.8.2.3074
+Installers:
+- InstallerLocale: en-US
+  Architecture: neutral
+  InstallerType: nullsoft
+  InstallerUrl: http://pascalabc.net/downloads/PascalABCNETMiniSetup.exe
+  InstallerSha256: FCE97894999A183C07AD845B770952ABBAD9BE501B84AD7E3A17CD72DC0B57E3
```
Comment (Trenly) [inline manifests/p/PascalABCNET/PascalABCNET-stable/3.8.2.3074/PascalABCNET.PascalABCNET-stable.installer.yaml:11]:
```suggestion
  InstallerSha256: FCE97894999A183C07AD845B770952ABBAD9BE501B84AD7E3A17CD72DC0B57E3
  ProductCode: PascalABCNET
  AppsAndFeaturesEntries: 
    - DisplayName: PascalABC.NET
      DisplayVersion: null
      Publisher: null
      ProductCode: PascalABCNET
```
ProductCode will be needed for package matching

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/p/PascalABCNET/PascalABCNET-stable/3.8.2.3074/PascalABCNET.PascalABCNET-stable.installer.yaml (hunk 4 above)
Comment (BareGamer) [reply manifests/p/PascalABCNET/PascalABCNET-stable/3.8.2.3074/PascalABCNET.PascalABCNET-stable.installer.yaml:11]: What is DisplayVersion and why is it set to null? Is ProductCode used for the Match field?

Comment (Trenly) [reply manifests/p/PascalABCNET/PascalABCNET-stable/3.8.2.3074/PascalABCNET.PascalABCNET-stable.installer.yaml:11]: DisplayVersion is the version shown in Control Panel on Windows 10

File: manifests/p/PascalABCNET/PascalABCNET-stable/3.8.2.3074/PascalABCNET.PascalABCNET-stable.locale.ru-Ru.yaml [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+PackageIdentifier: PascalABCNET.PascalABCNET
+PackageVersion: 3.8.2.3074
+PackageLocale: ru-Ru
+Publisher: PascalABC Compiler Team
+Author: PascalABC Compiler Team
+PackageName: PascalABC.NET
+PackageUrl: http://pascalabc.net/downloads
+License: GNU GPL v3
+ShortDescription: PascalABC IDE website release
```
Comment (ItzLevvie) [inline manifests/p/PascalABCNET/PascalABCNET-stable/3.8.2.3074/PascalABCNET.PascalABCNET-stable.locale.ru-Ru.yaml:12]:
```suggestion
ShortDescription: The new generation Pascal programming language that combines simplicity of classic Pascal, a great number of modern extensions and broad capabilities of Microsoft .NET Framework.
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request closed without merging**